Mathlib/Data/Finsupp/Basic.lean

```diff
@@ -1691,26 +1691,72 @@ instance uniqueOfLeft [IsEmpty α] : Unique (α →₀ R) :=
 
 end
 
+section
+variable {M : Type*} [Zero M] {P : α → Prop} [DecidablePred P]
+
+/-- Extend the domain of a `Finsupp` by using `0` where `P x` does not hold. -/
+@[simps]
+def extendDomain (f : Subtype P →₀ M) : α →₀ M where
```

Comment:
NB. At first, I would have expected an abbreviation such as extDomain, to stay close to embDomain, but the name seems in par with Equiv.Perm.extendDomain.